recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,49 @@
+c_compiler:                    # [win]
```

Comment:
This file would not live in this feedstock. It would be fetched from a common place and populated with all the pinnings from `pin_slow_way.py`.

Reply:
Yes, exactly.